app/views/layouts/application.html.erb

```diff
@@ -50,6 +50,12 @@
               <button id="butInstall" class="material-icons mdc-top-app-bar__action-item mdc-icon-button">cloud_download</button>
             </span>
             <%= yield :credits %>
+            <div class="mdc-menu-surface--anchor" data-controller='user-menu'>
+              <button class="material-icons mdc-top-app-bar__action-item mdc-icon-button" aria-label="Open user menu" data-user-menu-target='trigger' data-action='click->user-menu#open'>
```

Comment:
are we using `aria` attributes or should we remove them?

Reply:
I mean for sure we are not using them in the rest of the application, but I though that it didn't hurt to have them. I guess I can remove them and maybe add a ticket in the backlog to review the application's accessibility. What do you think?

Reply:
Updated! [52108bc](https://github.com/cedarcode/mi_carrera/pull/300/commits/52108bc4dff24adca6bf81b9a3122ee30cf833cb)